src/components/home/home-lazy.ts

```diff
@@ -0,0 +1,8 @@
+import { FC, lazy, LazyExoticComponent } from 'react';
+
+import { routeWithSuspense } from '../../route-with-suspense';
+import Loader from '../loader/loader';
+
+const HomeLazy: LazyExoticComponent<FC> = lazy(() => import('./home'));
```

Comment:
No need to specify such file type
```suggestion
const HomeLazy = lazy(() => import('./home'));
```